src/recipes/shembot.json

```diff
@@ -0,0 +1,24 @@
+{
+    "name": "Shembot",
+    "description": "Shembot a Filipino Legacy drink of all time Filipino favorite low budget drink for small time occasion or parties",
+    "github": "joamarderije",
+    "ingredients": [
+        "1 Bottle of Ginebra San Miguel ",
+        "1 25g Pack of Melon Juice Flavor",
+        "1 25g pack of Coffee",
+        "a pack of cube ice"
+    ],
+    "directions": [
+        "1.get a pitcher filled with water(3/4).",
+        "2.pour the Gin into the pitcher together with the melon juice and nescafe.",
+        "3.after pouring the gin in the pitcher, light the bottle using a match or a lighter and subside bottle in the pitcher",
+        "4. serve it with ice ;)"
```

Comment:
Hi, the numeration can be removed as this list is rendered as an ordered list and will display numbers already.